### PR TITLE
Add WebSphere Liberty distribution

### PR DESCRIPTION
**Pull request by hibell**
## Summary
This PR allows users to configure the buildpack to install the WebSphere Liberty app server.

## Use Cases
Allow users to install WLP by setting `BP_LIBERTY_INSTALL_TYPE` to `wlp`.

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [X] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,16 +36,37 @@ The buildpack will support all available profiles of the most recent versions of
...
+| `$BP_JAVA_APP_SERVER`      | The application server to use. It defaults to `` (empty string) which means that order dictates which Java application server is installed. The first Java application server buildpack to run will be picked.                                                                                                                                                                                      |
+| `$BP_LIBERTY_INSTALL_TYPE` | [Install type](#install-types) of Liberty. Valid options: `ol`, `wlp`, and `one`. Defaults to `ol`.                                                                                                                                                                                                                                                                                                 |
```
Comment (kevin-ortega) [inline README.md:42]: `one` should be `none`?

File: README.md [hunk 2]
```diff
@@ -36,16 +36,37 @@ The buildpack will support all available profiles of the most recent versions of
...
+| `$BP_LIBERTY_VERSION`      | The version of Liberty to install. Defaults to the latest version of the runtime. To see what version is available with your version of the buildpack, please see the [release notes](https://github.com/paketo-buildpacks/liberty/releases). At present, only the latest version is supported, and you need to use an older version of the buildpack if you want an older version of Open Liberty. |
```
Comment (kevin-ortega) [inline README.md:43]: should probably indicate Open Liberty or WebSphere Liberty, or simply Liberty.

File: buildpack.toml [hunk 3]
```diff
@@ -198,6 +198,76 @@ api = "0.7"
...
+    name = "WebSphere Liberty (Jakarta EE9)"
+    purl = "pkg:maven/com.ibm.websphere.appserver.runtime/wlp-jakartaee9@22.0.0.8"
+    sha256 = "ef0c9b2eba2eeadbca565e64ae0b99cecb6d06f0a0eddd0b892904b50dccde7b"
+    stacks = ["io.buildpacks.stacks.bionic", "*"]
+    uri = "https://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-kernel/22.0.0.8/wlp-jakartaee9-22.0.0.8.zip"
+    version = "21.0.8"
+
+    [[metadata.dependencies.licenses]]
+      type = "Proprietary"
+      uri = "https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/license/base_ilan/ilan/22.0.0.8/lafiles/en.html"
```
Comment (kevin-ortega) [inline buildpack.toml:227]: Use https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/22.0.0.8/lafiles/runtime/en.html

File: buildpack.toml [hunk 4]
```diff
@@ -198,6 +198,76 @@ api = "0.7"
...
+    name = "WebSphere Liberty (Java EE8)"
+    purl = "pkg:maven/com.ibm.websphere.appserver.runtime/wlp-javaee8@22.0.0.8"
+    sha256 = "ef0c9b2eba2eeadbca565e64ae0b99cecb6d06f0a0eddd0b892904b50dccde7b"
+    stacks = ["io.buildpacks.stacks.bionic", "*"]
+    uri = "https://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-javaee8/22.0.0.8/wlp-javaee8-22.0.0.8.zip"
+    version = "21.0.8"
+
+    [[metadata.dependencies.licenses]]
+      type = "Proprietary"
+      uri = "https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/license/base_ilan/ilan/22.0.0.8/lafiles/en.html"
```
Comment (kevin-ortega) [inline buildpack.toml:241]: use https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/22.0.0.8/lafiles/runtime/en.html

File: buildpack.toml [hunk 5]
```diff
@@ -198,6 +198,76 @@ api = "0.7"
...
+    name = "WebSphere Liberty (Web Profile 9)"
+    purl = "pkg:maven/com.ibm.websphere.appserver.runtime/wlp-webProfile9@22.0.0.8"
+    sha256 = "ef0c9b2eba2eeadbca565e64ae0b99cecb6d06f0a0eddd0b892904b50dccde7b"
+    stacks = ["io.buildpacks.stacks.bionic", "*"]
+    uri = "https://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-webProfile9/22.0.0.8/wlp-webProfile9-22.0.0.8.zip"
+    version = "21.0.8"
+
+    [[metadata.dependencies.licenses]]
+      type = "Proprietary"
+      uri = "https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/license/base_ilan/ilan/22.0.0.8/lafiles/en.html"
```
Comment (kevin-ortega) [inline buildpack.toml:255]: Use https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/22.0.0.8/lafiles/runtime/en.html

File: buildpack.toml [hunk 6]
```diff
@@ -198,6 +198,76 @@ api = "0.7"
...
+    name = "WebSphere Liberty (Web Profile 8)"
+    purl = "pkg:maven/com.ibm.websphere.appserver.runtime/wlp-webProfile8@22.0.0.8"
+    sha256 = "ef0c9b2eba2eeadbca565e64ae0b99cecb6d06f0a0eddd0b892904b50dccde7b"
+    stacks = ["io.buildpacks.stacks.bionic", "*"]
+    uri = "https://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-webProfile8/22.0.0.8/wlp-webProfile8-22.0.0.8.zip"
+    version = "21.0.8"
+
+    [[metadata.dependencies.licenses]]
+      type = "Proprietary"
+      uri = "https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/license/base_ilan/ilan/22.0.0.8/lafiles/en.html"
```
Comment (kevin-ortega) [inline buildpack.toml:269]: Use https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/22.0.0.8/lafiles/runtime/en.html

File: liberty/build.go [hunk 7]
```diff
@@ -239,3 +279,26 @@ func createStackRuntimeProcess(serverName string) (libcnb.Process, error) {
...
+			profile == "kernel" ||
+			profile == "jakartaee9" ||
+			profile == "javaee8" ||
+			profile == "webProfile9" ||
+			profile == "webProfile8" ||
+			profile == "microProfile5" ||
+			profile == "microProfile4"
+	}
+
+	if distType == websphereLibertyInstall {
```
Comment (kevin-ortega) [inline liberty/build.go:295]: Any reason you didn't include webProfile7, javaee7, or javaee9 for WLP?

File: buildpack.toml (hunk 6 above)
Comment (hibell) [reply buildpack.toml:269]: Thanks.

**Review by kevin-ortega — APPROVED** [review]
I'm good with these changes.   Thanks Hank.

**Review by dmikusa-pivotal — APPROVED** [review]

**Pull request merged**